spec/amber/scripts/environment_loader_spec.cr

```diff
@@ -0,0 +1,47 @@
+require "../../../spec_helper"
+
+describe Amber::Server do
```

Comment:
I like the idea of moving this under scripts. Would it be possible to move it and maintain git file history though instead of adding a new file?

Reply:
I can undo the commit, and you can go ahead and move it in a separate PR. WDYT?